src/prepack-node.js

```diff
@@ -17,12 +17,11 @@ import { InitializationError } from "./prepack-standalone";
 import { prepackNodeCLI, prepackNodeCLISync } from "./prepack-node-environment";
 
 import type { Options } from "./options";
-import { defaultOptions } from "./options";
 
 export * from "./prepack-standalone";
 export * from "./prepack-node-environment";
 
-export function prepackStdin(options: Options = defaultOptions, callback: Function) {
+export function prepackStdin(options: Options, callback: Function) {
```

Comment:
I don't think that requiring people to pass in {} explicitly amounts to real hardship. I like the idea of reminding them at every call to think about the options they want to use.